README.md

```diff
@@ -37,37 +37,29 @@ npm install osprey-method-handler --save
 ## Usage
 
 ```js
-var express = require('express')
-var handler = require('osprey-method-handler')
-var app = express()
-
-app.post('/users', handler({
-  headers: {},
-  responses: {
-    '200': {
-      body: {
-        'application/json': {
-          schema: '...',
-          example: '...'
-        }
-      }
-    }
-  },
-  body: {
-    'application/json': {
-      schema: '...'
-    }
+const express = require('express')
+const handler = require('osprey-method-handler')
+const utils = require('./utils')
+
+const app = express()
+
+// webapi-parser.Operation
+const methodObj = utils.getMethodObj()
+const options = {}
+
+app.post(
+  '/users',
+  handler(methodObj, '/users', 'POST', options),
+  function (req, res) {
+    res.send('success')
   }
-}, '/users', 'POST', { /* ... */ }), function (req, res) {
-  res.send('success')
-})
+)
 ```
-
-Accepts the RAML schema as the first argument, method and path in subsequent arguments (mostly for debugging) and options as the final argument.
+Accepts [webapi-parser](https://github.com/raml-org/webapi-parser) `Operation` object as first argument, path string as second argument, method name as third and options object as final argument.
 
 **Options**
 
-* `ajv` Custom [Ajv](https://github.com/epoberezkin/ajv) instance to be used for JSON validation
+* `ajv` Custom [Ajv](https://github.com/epoberezkin/ajv) instance to be used for validation
```

Comment:
Why remove the word `JSON`? Is it because we can now use AJV validation for any types? Does that mean XML too? If not, I would list the types that are being supported, e.g. JSON Schema drafts 3-?, RAML 1.0 Datatypes, OAS Schema, ...

Reply:
Previously ajv was only used to validate JSON request bodies. Now it's used everywhere except XML request bodies validation. I've just clarified the option description.